src/main/java/org/mitre/synthea/export/FhirStu3.java

```diff
@@ -810,6 +820,313 @@ private static BundleEntryComponent encounterClaim(BundleEntryComponent personEn
+      p.setDate(proc.getDate());
+      p.setSequence(proc.getSequence());
+      p.setProcedure(proc.getProcedure());
+    }
+
+    List<ExplanationOfBenefit.ItemComponent> eobItem = new ArrayList<>();
+
+    double totalPayment = 0;
+    // Get all the items info from the claim
+
+    for (ItemComponent item : claim.getItem()) {
+
+      ExplanationOfBenefit.ItemComponent itemComponent = new ExplanationOfBenefit.ItemComponent();
+
+      itemComponent.setSequence(item.getSequence());
+      itemComponent.setCareTeamLinkId(item.getCareTeamLinkId());
+      itemComponent.setDiagnosisLinkId(item.getDiagnosisLinkId());
+      itemComponent.setInformationLinkId(item.getInformationLinkId());
+      itemComponent.setNet(item.getNet());
+      itemComponent.setQuantity(item.getQuantity());
+      itemComponent.setUnitPrice(item.getUnitPrice());
+      itemComponent.setEncounter(item.getEncounter());
+
+      // can probably set code based on encounter type with switch statement
+      itemComponent.setService(new CodeableConcept().addCoding(new Coding()
+          .setSystem("https://bluebutton.cms.gov/resources/codesystem/hcpcs")
+          .setCode("99212") //99211 to 99215 are outpatient codes based on severity
+          .setDisplay("d")));
+      itemComponent.setServiced(encounterResource.getPeriod());
+      // Location of service, can use switch statement based on
+      // encounter type
+      String code;
+      String display;
+      CodeableConcept location = new CodeableConcept();
+      switch (encounter.type) {
+        case Provider.AMBULATORY:
+          code = "21";
+          display = "Inpatient Hospital";
+          break;
+        case Provider.EMERGENCY:
+          code = "23";
+          display = "Emergency Room";
+          break;
+        case Provider.INPATIENT:
+          code = "21";
+          display = "Inpatient Hospital";
+          break;
+        case Provider.URGENTCARE:
+          code = "20";
+          display = "Urgent Care Facility";
+          break;
+        case Provider.WELLNESS:
+          code = "22";
+          display = "Outpatient Hospital";
+          break;
+        default:
+          code = "21";
+          display = "Inpatient Hospital";
+      }
+      location.addCoding()
+          .setCode(code)
+          //.setSystem("http://hl7.org/fhir/ValueSet/service-place") > if we wanted hl7
+          .setSystem("https://bluebutton.cms.gov/resources/variables/line_place_of_srvc_cd")
+          .setDisplay(display);
+      itemComponent.setLocation(location);
+
+
+      itemComponent.setCategory(new CodeableConcept().addCoding(new Coding()
+          .setSystem("https://bluebutton.cms.gov/resources/variables/line_cms_type_srvc_cd")
+          .setCode("1")
+          .setDisplay("Medical care")));
+
+
+
+      // Adjudication
+      if (item.hasNet()) {
+
+        // Assume that the patient has already paid deductible and
```

Comment:
Technically, the patient might not have insurance... in which case there is no deductible or coinsurance. Mark this section with a TODO comment.

```java
// TODO: Change deductibles, coinsurance, and payment amounts based on insurance.
```